docker-compose.yml

```diff
@@ -24,8 +24,11 @@ boulder:
       - 9380:9380 # mail-test-srv
       - 9381:9381 # mail-test-srv
     links:
+      - bhsm:boulder-hsm
       - bmysql:boulder-mysql
       - brabbitmq:boulder-rabbitmq
+bhsm:
+    image: letsencrypt/boulder-hsm:2016-07-08
```

Comment:
Add a comment: "To minimized fetches, this should match the FROM image in Dockerfile."